src/common.cpp

```diff
@@ -248,4 +248,34 @@ void Renderable::transform_translate(vec2 offset)
 void Renderable::transform_end()
 {
 	//
+}
+
+char* get_month_from_index(int month_index)
+{
+	switch (month_index) {
+	case 0:
+		return "January";
+	case 1:
+		return "February";
+	case 2:
+		return "March";
+	case 3:
+		return "April";
+	case 4:
+		return "May";
+	case 5:
+		return "June";
+	case 6:
+		return "July";
+	case 7:
+		return "August";
+	case 8:
+		return "September";
+	case 9:
+		return "October";
+	case 10:
+		return "November";
+	case 11:

```

Comment:
build complained about not having a default case for this.

Reply:
Addressed